cmd/ui/src/views/Explore/ContextMenu/CopyMenuItem.tsx

```diff
@@ -67,11 +68,13 @@ const CopyMenuItem = () => {
                 title={
                     <>
                         <MenuItem onClick={handleCopyDisplayName}>Display Name</MenuItem>
-                        <MenuItem onClick={handleCopyObjectId}>Object ID</MenuItem>
-                        <MenuItem onClick={handleCopyCypher}>Cypher</MenuItem>
+                        {isNodeSelected && <MenuItem onClick={handleCopyObjectId}>Object ID</MenuItem>}
+                        {isNodeSelected && <MenuItem onClick={handleCopyCypher}>Cypher</MenuItem>}
                     </>
                 }>
-                <MenuItem sx={{ justifyContent: 'space-between' }} onClick={(e) => e.stopPropagation()}>
+                <MenuItem
+                    sx={{ justifyContent: 'space-between', minWidth: '8rem' }}
```

Comment:
`min-width` ensures that the submenu arrow isn't right up against the Copy when no other options are shown.